src/store/albums.js

```diff
@@ -1,34 +1,49 @@
 import Vue from "vue";
-import { create, destroy, index, update, destroyEmpty } from "../api/albums";
+import { create, destroy, update, destroyEmpty } from "../api/albums";
+import { fetchAll } from "./commit";
 import { compareStrings } from "../comparators";
 
 export default {
   namespaced: true,
   state: {
     albums: {},
+    startLoading: new Date(0),
```

Comment:
I feel like this should not be in the state, but something kept in the (generic) index function. It can then be given as a parameter to `removeOld`.

Reply:
I've put this in the state, so that we can use this value (for example, to disable edit as discussed above)

Reply:
(also, we need this per module, since we don't always load all modules at the same time)

Reply:
I fail to see how this would help to disable edit. For that you would need a "loading" property, not the time you started loading.

Reply:
A loading property would either disable editing on all albums or none. 
My intention was to compare startLoading to loaded prop on the album. That way we can re-enable editing on an individual album from the moment it is reloaded (even while other albums are still being loaded). Thus, we disable for the shortest time possible.

Reply:
Hmm, didn't think about that. Sure, no problem then.